### PR TITLE
two column block fe

### PR DESCRIPTION
**Pull request by hippogriffic**

### CODE REVIEW

File: django-verdant/rca/static/rca/css/variables.less [hunk 1]
```diff
@@ -152,6 +152,7 @@ you would use 'font-weight:bold;', use the '.bold()' mixin instead */
 @color-warm-grey: #b6a2a2;
 @color-middle-column-lines: #d6d6d6;
 @color-tab-grey: darken(@color-pale-grey, 10%);
+@color-icon-grey: #cdcdcd;
```
Comment (helenb) [inline django-verdant/rca/static/rca/css/variables.less]: Might this be close enough to the existing `@color-light-grey` to use that instead?

File: django-verdant/rca/static/rca/css/streamfield-blocks/two-column-block.less [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.two-column-block {
+    @media @media-desktop-small {
+        display: flex;
+    }
+
+    &__item {
+        flex-basis: 50%;
```
Comment (helenb) [inline django-verdant/rca/static/rca/css/streamfield-blocks/two-column-block.less]: mobile styling will have these stretch all the way across won't it?

**Comment by helenb** [comment]
Just a couple of minor points. I think we should probably stick to the existing colours as far as possible.

File: django-verdant/rca/static/rca/css/variables.less (hunk 1 above)
Comment (hippogriffic) [reply django-verdant/rca/static/rca/css/variables.less]: removed

File: django-verdant/rca/static/rca/css/streamfield-blocks/two-column-block.less (hunk 2 above)
Comment (hippogriffic) [reply django-verdant/rca/static/rca/css/streamfield-blocks/two-column-block.less]: this won't have any effect until `display: flex` is set at 768px

**Pull request merged**